src/server.h

```diff
@@ -2027,6 +2027,7 @@ struct valkeyServer {
     unsigned long long cluster_link_msg_queue_limit_bytes;  /* Memory usage limit on individual link msg queue */
     int cluster_drop_packet_filter; /* Debug config that allows tactically
                                    * dropping packets of a specific type */
+    sds cached_cluster_slot_info[CACHE_CONN_TYPE_MAX];
```

Comment:
Any reason to maintain this state here instead of clusterState?

Reply:
Cluster state is implementation specific, (so cluster v1 vs cluster v2), and the cached information is agnostic to the underlying implementation.